api/Document.json

```diff
@@ -912,7 +912,7 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "2",
+              "version_added": "1",
               "notes": "The <code>body</code> property was implemented on the <code>HTMLDocument</code> interface in Firefox for a long time, hence <code>document.body</code> would not return the <code>&lt;body&gt;</code> element if the document's <code>Content-Type</code> was not set to <code>text/html</code> or <code>application/xhtml+xml</code> (or if it came from <code>DOMParser.parseFromString</code> without the <code>text/html</code> type being used). This has been fixed in Firefox 60."
```

Comment:
I filed https://github.com/mdn/browser-compat-data/issues/7517 about this note :)